node/src/BaseClient.ts

```diff
@@ -3416,6 +3418,39 @@ export class BaseClient {
         );
     }
 
+    /**
+     * Pops a member-score pair from the first non-empty sorted set, with the given `keys`
+     * being checked in the order they are provided.
+     *
+     * See https://valkey.io/commands/zmpop/ for more details.
+     *
+     * @remarks When in cluster mode, all `keys` must map to the same hash slot.
+     * @param keys - The keys of the sorted sets.
+     * @param modifier - The element pop criteria - either {@link ScoreFilter#MIN} or
+     *     {@link ScoreFilter#MAX} to pop the member with the lowest/highest score accordingly.
```

Comment:
```suggestion
     * @param modifier - The element pop criteria - either {@link ScoreFilter.MIN} or
     *     {@link ScoreFilter.MAX} to pop the member with the lowest/highest score accordingly.
```
Please double check link format to ensure they are clickable. Update in transaction too.